### PR TITLE
Feat/remove tx

### PR DESCRIPTION
**Pull request by matiu**
adds remove TX

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -482,6 +481,46 @@ CopayServer.prototype.removeWallet = function(opts, cb) {
...
+ * @param {string} opts.id - The tx id.
+ * @return {undefined}
+ */
+CopayServer.prototype.removePendingTx = function(opts, cb) {
+  var self = this;
+
+  if (!Utils.checkRequired(opts, ['id']))
+    return cb(new ClientError('Required argument missing'));
+
+  Utils.runLocked(self.id, cb, function(cb) {
```
Comment (isocolsky) [inline lib/server.js]: did you mean to lock on ```self.walletId```or ```opts.id```?

File: test/integration.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ helpers.createAndJoinWallet = function(m, n, cb) {
 
       helpers.getAuthServer(copayerIds[0], function(s) {
         s.getWallet({}, function(err, w) {
-          cb(s, w, _.take(TestData.copayers, w.n));
+          cb(s, w, _.take(TestData.copayers, w.n), copayerIds);
```
Comment (isocolsky) [inline test/integration.js]: the 3rd argument is not really needed, we could always assume the wallet contains TestData.copayers[0...n]

File: lib/server.js (hunk 1 above)
Comment (matiu) [inline lib/server.js]: thanks. self.walletId. Fixed.

Comment (matiu) [inline lib/server.js]: maybe we should not use `id`, but always walletId / txProposalId/ and so.

**Pull request merged**